scripts/algcomptool/tool/AlgCompTool.py

```diff
@@ -0,0 +1,317 @@
+#Author: Mihai Avram - mihai.v.avram@gmail.com
+
+import sys, getopt
+import pandas as pd
+import numpy as np
+import string
+
+    
+def main(argv):
+    if len(argv) < 2:
+        print("Executing...\n")
+        computeAndPrint(False)
+    elif(argv[1] in ['--help','-h']):
+        print("Enter python AlgCompTool.py to print to terminal, or python AlgCompTool.py -f to print to files")
+    elif(argv[1] in ['--file','-f']):
+        print("Executing...\n")
+        computeAndPrint(True)
+
+
+def computeAndPrint(writeToFile):
+    #Importing algorithms to be compared
+    alg1 = pd.read_csv("../input/presidentcouplesNODES.csv")
+    alg2 = pd.read_csv("../input/presidentcouplesRSIM.csv")
+
+    noNulls = 1
+    #Alg1 checking for null values in pertinent columns
+    if any(pd.isnull(alg1['sid'])) or any(pd.isnull(alg1['oid'])) or any(pd.isnull(alg1['class'])) or any(pd.isnull(alg1['score'])):
+        noNulls = 0
+        print("There are missing column values in the sid, oid, class, or score columns in alg1")
+
+    #Alg2 checking for null values in pertinent columns
+    if any(pd.isnull(alg2['sid'])) or any(pd.isnull(alg2['oid'])) or any(pd.isnull(alg2['class'])) or any(pd.isnull(alg2['score'])):
+        noNulls = 0
+        print("There are missing column values in the sid, oid, class, or score columns in alg2")
+    
+
+    #Algorithm comparison
+    if noNulls:
```

Comment:
Instead of enclosing all computations in a if block, exit if there are nulls (e.g. `sys.exit(1)` to return error status code), and then continue the rest of the computation without the if block:

```
if there_is_an_error:
    print("The was an error", file=sys.stderr)
    sys.exit(1) 

<NORMAL OPERATIONS>
```